app/views/lti_nrps/sync_roster.html.erb

```diff
@@ -1,26 +1,2 @@
 <h2>Sync Course Roster</h2>
-<div class="table-info">
-  <div>
-    Existing accounts in <b>Black</b>.  New accounts in <b><font
-    color=green>Green</font></b>.  Dropped accounts in <b><font
-    color=red>Red</font></b>. Instructors are not shown. <br>
-  </div>
-  <div>
-    <label>
-      <input type="checkbox" id="sort-status" name="sort-status" class="filled-in" checked="checked" />
-      <span>Sort by account status</span>
-    </label>
-  </div>
-</div>
-<%= form_for :doIt, url: upload_roster_course_path(name: @course.name, doIt: "true") do |f| %>
-  <%= f.submit "Confirm", {:class => "btn primary"} %>
-  <br>
-  <%= render partial: 'courses/uploadTable', locals: {cud_view:@cuds, sort_status:"unsorted_cuds"} %>
-  <%= render partial: 'courses/uploadTable', locals: {cud_view:@sorted_cuds, sort_status:"sorted_cuds"} %>
-  <br>
-  <%= f.submit "Confirm", {:class => "btn primary"} %>
-<% end %>
-
-<% content_for :javascripts do %>
-<%= javascript_include_tag "sort-toggle" %>
-<% end %>
\ No newline at end of file
+<%= render partial: 'courses/updateRosterPreview', locals: { cuds:@cuds, sorted_cuds: @sorted_cuds} %>
```

Comment:
Nit: Add space after `cuds:`. Same for `upload_roster.html.erb`

Reply:
Will add the spaces